geo/src/algorithm/map_coords.rs

```diff
@@ -583,6 +813,27 @@ impl<T: CoordNum> MapCoordsInplace<T> for Triangle<T> {
     }
 }
 
+impl<T: CoordNum, E> TryMapCoordsInplace<T, E> for Triangle<T> {
+    fn try_map_coords_inplace(
+        &mut self,
+        func: impl Fn(&(T, T)) -> Result<(T, T), E>,
```

Comment:
This is another artifact of history - but I'm curious - based on your usage, is it actually convenient to deal with (T, T) tuples here rather than Coordinate<T>?

I think it makes sense to be consistent with all the other ones for now, but maybe something to consider for a future  breaking change or alternate API.

Reply:
My preference is always to prefer working with structures with named fields when working with coordinates, as opposed to unnamed positional arguments. So yes, I do think it would be an improvement to change the tuple here to `Coordinate`. In fact we'll probably need to do that if/when the XYZM coordinate changes come in.

Reply:
I have a funny feeling that I'm responsible for this API, and that it's been somewhat inconvenient since the day it was introduced.